satellite/marketingweb/server.go

```diff
@@ -171,6 +173,9 @@ func (s *Server) CreateOffer(w http.ResponseWriter, req *http.Request) {
 		offer.Type = rewards.Referral
 	case "free-credit":
 		offer.Type = rewards.FreeCredit
+	case "partner-offer":
+		offer.Type = rewards.Partner
+		offer.Name = offer.FormatPartnerName()
```

Comment:
Do we really need to do this? Since the `name` field is readonly anyway, we won't run into the situation where there are two identical partner names